src/library/scala/collection/immutable/NumericRange.scala

```diff
@@ -313,14 +368,14 @@ sealed class NumericRange[T](
   }
 
   override lazy val hashCode: Int = super.hashCode()
-  override protected final def applyPreferredMaxLength: Int = Int.MaxValue
+  final override protected def applyPreferredMaxLength: Int = Int.MaxValue
 
   override def equals(other: Any): Boolean = other match {
     case x: NumericRange[_] =>
       (x canEqual this) && (length == x.length) && (
-        (isEmpty) ||                            // all empty sequences are equal
-          (start == x.start && last == x.last)  // same length and same endpoints implies equality
-        )
+        (isEmpty) || // all empty sequences are equal
```

Comment:
Easier to visually separate comments and code if the comments are aligned with each other